js/understanding-hoisting-the-real-way/understanding-hoisting-the-real-way.md

```diff
@@ -0,0 +1,185 @@
+Hello, I am Shivam Verma. Today we are going to understand one of the most frequently asked questions in interviews. 
+What is **Hoisting** in JavaScript? 
+
+First of all, let us understand the concept of **Hoisting**. Let's Begin
+
+## Understanding "Hoisting"
+
+Let us take a look at a code example.
```

Comment:
Keep it short and precise, always to say small sentences without losing the meaning. Example: Let's take an example

We still said the same thing but in less words